x-pack/plugins/security_solution/public/management/hooks/endpoint/use_get_endpoint_policy_response.tsx

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+import { HttpFetchError, HttpSetup } from '@kbn/core/public';
+import { QueryObserverResult, useQuery, UseQueryOptions } from 'react-query';
+import { BASE_POLICY_RESPONSE_ROUTE } from '../../../../common/endpoint/constants';
+import { GetHostPolicyResponse } from '../../../../common/endpoint/types';
+
+export function useGetEndpointPolicyResponse(
+  http: HttpSetup,
+  selectedEndpoint: string,
+  customQueryOptions?: UseQueryOptions<GetHostPolicyResponse, HttpFetchError>
+): QueryObserverResult<GetHostPolicyResponse, HttpFetchError> {
```

Comment:
`QueryObserverResult<>` should probably be `UseQueryResponse<>`

Reply:
Did you meant `UseQueryResult` ?

Reply:
Sorry. yes, `UseQueryResult<>` - that's the type that `useQuery()` returns